File: answers/Go/@codehakase/13-tax-classifier.go
package main

import "fmt"

func main() {
	var salary float64
	for {
		fmt.Print("Enter salary amount: ")
		fmt.Scanf("%f", &salary)
		if salary < 1 {
			fmt.Println("Invalid salary amount")
		}
		if salary < 50000 {
			conv := salary * 5 / 100
			fmt.Printf("Tax: %.2f\n", conv)
		} else if salary <= 200000 {
			conv := salary * 10 / 100
			fmt.Printf("Tax: %.2f\n", conv)
		} else {
			conv := salary * 15 / 100
			fmt.Printf("Tax: %.2f\n", conv)
		}
	}
}

File: programming/answers/C/@mykeels/13-tax-classifier.c
#include <stdio.h>

int main() {
    while (1) {
        printf("Enter a salary amount: ");

        double salary = 0;
        scanf("%lf", &salary);

        if (salary < 0) {
            printf("Invalid Amount");
        }
        else if (salary < 50000) {
            printf("Tax %f\n", salary * 5 / 100);
        }
        else if (salary < 200000) {
            printf("Tax %f\n", salary * 10 / 100);
        }
        else {
            printf("Tax %f\n", salary * 15 / 100);
        }
    }
    return 0;
}